python-sdk/tutorials/automl-with-azureml/forecasting-many-models/auto-ml-forecasting-many-models.ipynb

```diff
@@ -414,22 +419,30 @@
     "from azureml.train.automl.runtime._many_models.many_models_parameters import (\n",
     "    ManyModelsTrainParameters,\n",
     ")\n",
+    "from azureml.automl.core.forecasting_parameters import ForecastingParameters\n",
+    "from azureml.automl.core.automl_base_settings import AutoMLBaseSettings\n",
     "\n",
     "partition_column_names = [\"Store\", \"Brand\"]\n",
-    "automl_settings = {\n",
-    "    \"task\": \"forecasting\",\n",
-    "    \"primary_metric\": \"normalized_root_mean_squared_error\",\n",
-    "    \"iteration_timeout_minutes\": 10,  # This needs to be changed based on the dataset. We ask customer to explore how long training is taking before settings this value\n",
-    "    \"iterations\": 15,\n",
-    "    \"experiment_timeout_hours\": 0.25,\n",
-    "    \"label_column_name\": \"Quantity\",\n",
-    "    \"n_cross_validations\": 3,\n",
-    "    \"time_column_name\": \"WeekStarting\",\n",
-    "    \"drop_column_names\": \"Revenue\",\n",
-    "    \"forecast_horizon\": 6,\n",
-    "    \"time_series_id_column_names\": partition_column_names,\n",
-    "    \"track_child_runs\": False,\n",
-    "}\n",
+    "\n",
+    "forecasting_parameters = ForecastingParameters(\n",
+    "    time_column_name=\"WeekStarting\",\n",
+    "    drop_column_names=\"Revenue\",\n",
+    "    forecast_horizon=6,\n",
+    "    time_series_id_column_names=partition_column_names,\n",
+    ")\n",
+    "\n",
+    "automl_settings = AutoMLBaseSettings(\n",
+    "    task_type=\"regression\",\n",
```

Comment:
The task type should be forecasting

Reply:
updated to use AutoMLConfig instead of AutoMLBaseSettings.